packages/cursorless-engine/src/actions/CutToClipboard.ts

```diff
@@ -21,35 +21,19 @@ export class CutToClipboard implements SimpleAction {
         const { editor, contentRange } = target;
         const removalHighlightRange = target.getRemovalHighlightRange();
 
-        if (target.isLine) {
-          return [
-            {
-              editor,
-              range: toCharacterRange(contentRange),
-              style: FlashStyle.referenced,
-            },
-            {
-              editor,
-              range: toLineRange(removalHighlightRange),
-              style: FlashStyle.pendingDelete,
-            },
-          ];
+        if (removalHighlightRange.type === "line") {
+          return getLineFlashDescriptors(
+            editor,
+            contentRange,
+            removalHighlightRange,
+          );
```

Comment:
Code here could be extracted into a function that just takes `target` to further reduce the levels of indentation/complexity